### ISSUE TITLE
Can't install snakemake in a fresh environment

### ISSUE THREAD

**Issue by iquasere** [bioconda/bioconda-recipes#25071]
Running ```conda create -name snakemake snakemake``` on my Windows 10 laptop with conda 4.8.4 gives
```
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed

UnsatisfiableError:
```
No information on what is causing incompatibilities, so how can I try to solve this?

**Comment by iquasere** [comment]
Sorry, I forgot there was the verbose option. Still can't find the problem tho
```
Solving environment: ...working... DEBUG conda.resolve:get_reduced_index(574): Retrieving packages for:
  - snakemake
DEBUG conda.resolve:filter_group(636): snakemake: pruned from 59 -> 0
DEBUG conda.core.solve:_add_specs(627): specs_map with targets: OrderedDict()
DEBUG conda.resolve:bad_installed(1117): Checking if the current environment is consistent
DEBUG conda.core.solve:_find_inconsistent_packages(493): inconsistent precs: None
DEBUG conda.resolve:gen_clauses(917): gen_clauses returning with clause count: 170668
DEBUG conda.resolve:generate_spec_constraints(923): generate_spec_constraints returning with clause count: 170668
DEBUG conda.common._logic:_run_sat(607): Invoking SAT with clause count: 170669
DEBUG conda.resolve:generate_spec_constraints(923): generate_spec_constraints returning with clause count: 170668
DEBUG conda.common._logic:_run_sat(607): Invoking SAT with clause count: 170669
DEBUG conda.core.solve:_run_sat(766): conflicting specs:
  - snakemake
DEBUG conda.core.solve:_run_sat(810): final specs to add:
  - snakemake
DEBUG conda.resolve:solve(1249): Solving for:
  - 0: snakemake target=None optional=False
DEBUG conda.resolve:solve(1259): Solve: Getting reduced index of compliant packages

Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed
Traceback (most recent call last):
  File "C:\Users\Utilizador\anaconda3\lib\site-packages\conda\exceptions.py", line 1079, in __call__
    return func(*args, **kwargs)
  File "C:\Users\Utilizador\anaconda3\lib\site-packages\conda\cli\main.py", line 84, in _main
    exit_code = do_call(args, p)
  File "C:\Users\Utilizador\anaconda3\lib\site-packages\conda\cli\conda_argparse.py", line 83, in do_call
    return getattr(module, func_name)(args, parser)
  File "C:\Users\Utilizador\anaconda3\lib\site-packages\conda\cli\main_create.py", line 41, in execute
    install(args, parser, 'create')
  File "C:\Users\Utilizador\anaconda3\lib\site-packages\conda\cli\install.py", line 292, in install
    raise e
  File "C:\Users\Utilizador\anaconda3\lib\site-packages\conda\cli\install.py", line 261, in install
    unlink_link_transaction = solver.solve_for_transaction(
  File "C:\Users\Utilizador\anaconda3\lib\site-packages\conda\core\solve.py", line 114, in solve_for_transaction
    unlink_precs, link_precs = self.solve_for_diff(update_modifier, deps_modifier,
  File "C:\Users\Utilizador\anaconda3\lib\site-packages\conda\core\solve.py", line 157, in solve_for_diff
    final_precs = self.solve_final_state(update_modifier, deps_modifier, prune, ignore_pinned,
  File "C:\Users\Utilizador\anaconda3\lib\site-packages\conda\core\solve.py", line 281, in solve_final_state
    ssc = self._run_sat(ssc)
  File "C:\Users\Utilizador\anaconda3\lib\site-packages\conda\common\io.py", line 88, in decorated
    return f(*args, **kwds)
  File "C:\Users\Utilizador\anaconda3\lib\site-packages\conda\core\solve.py", line 815, in _run_sat
    ssc.solution_precs = ssc.r.solve(tuple(final_environment_specs),
  File "C:\Users\Utilizador\anaconda3\lib\site-packages\conda\common\io.py", line 88, in decorated
    return f(*args, **kwds)
  File "C:\Users\Utilizador\anaconda3\lib\site-packages\conda\resolve.py", line 1285, in solve
    self.find_conflicts(specs, specs_to_add, history_specs)
  File "C:\Users\Utilizador\anaconda3\lib\site-packages\conda\resolve.py", line 352, in find_conflicts
    raise UnsatisfiableError(bad_deps, strict=strict_channel_priority)
conda.exceptions.UnsatisfiableError
```

**Comment by mbargull** [comment]
Hi @iquasere,
we offer two packages: `snakemake` and `snakemake-minimal`.
With `snakemake-minimal` you can install Snakemake itself (plus all dependencies that are required for its core functionality) on all platforms.
The `snakemake` package will also install `snakemake-minimal` but addtionally many other packages that are needed for extended functionalities of Snakemake. However, some of those packages are not available on Windows and as such some functionalities are not available with on that platform (e.g., IIRC, the "reports" draw graphs with `pygraphviz`, but that Conda package is not available on Windows).

TL;DR: You can install `snakemake-minimal` on Windows to get Snakemake's core functions. If you need certain extended functionalities, you'd have to install `snakemake` on Linux/macOS systems.

**Comment by iquasere** [comment]
@mbargull thank you for handling hundreds of issues of Anaconda per day ahah And for helping in this as well, it is solved using ```snakemake-minimal```
